std_msgs/Header header
uint8 rem_id
uint8 local_id
uint8 rssi
uint8 rem_rssi
uint8 noise
uint8 rem_noise
